src/ONYXKEYS.ts

```diff
@@ -1202,6 +1207,8 @@ type OnyxValuesMapping = {
     [ONYXKEYS.NVP_LAST_IPHONE_LOGIN]: string;
     [ONYXKEYS.NVP_LAST_ANDROID_LOGIN]: string;
     [ONYXKEYS.TRANSACTION_THREAD_NAVIGATION_REPORT_IDS]: string[];
+    [ONYXKEYS.REPORT_NAVIGATION_REPORT_IDS]: Record<string, string>;
```

Comment:
All usages of this onyx key that I can see are using an array format. Can't we just keep an array here?
```suggestion
    [ONYXKEYS.REPORT_NAVIGATION_REPORT_IDS]: string[];
```